test/test.js

```diff
@@ -140,8 +140,11 @@ describe('highlight:', function(argument) {
         } catch (err) {
           return code;
         }
-      }
+      },
     });
-    assert.equal(actual, '<h1>EEE</h1>\n<pre><code><span class="hljs-keyword">var</span> <span class="hljs-keyword">message</span> = <span class="hljs-string">\'This is an alert\'</span>;\nalert(<span class="hljs-keyword">message</span>);\n</code></pre>\n');
+    assert.equal(
+      actual,
+      '<h1>EEE</h1>\n<pre><code class="lang-javascript"><span class="hljs-keyword">var</span> message = <span class="hljs-string">\'This is an alert\'</span>;\n<span class="hljs-title function_">alert</span>(message);\n</code></pre>\n'
+    );
```

Comment:
The proper syntax for the code block is to specify a language after the triple back-ticks. If you remove `javascript` from `test/fixtures/e.md` this assert will pass without needing this modification, however you'll get a warning in your console notifying you that you forgot to pass in a language. Once I tested this and confirmed that it worked (without the `javascript`) I went ahead and added it and then updated the assertion.